### ISSUE TITLE
I can't find location. 

### ISSUE THREAD

**Issue by badaween-ahmed** [humazed/google_map_location_picker#15]
I just run the example app and I am getting these exceptions and can't get any location only Lat and Long.
```
I/flutter ( 3549): package:google_map_location_picker/src/map.dart 284:7 | GeolocationStatus.granted
I/flutter ( 3549): package:google_map_location_picker/src/map.dart 60:7 | position = Lat: ***, Long: ***
I/flutter ( 3549): package:google_map_location_picker/src/map.dart 284:7 | GeolocationStatus.granted
I/Google Maps Android API( 3549): Google Play services package version: 19420039
W/System  ( 3549): A resource failed to call release. 
I/flutter ( 3549): onCameraMoveStarted#_lastMapPosition = LatLng(45.521563, -122.677433)
I/flutter ( 3549): package:google_map_location_picker/src/map.dart 284:7 | GeolocationStatus.granted
I/flutter ( 3549): RangeError (index): Invalid value: Valid value range is empty: 0
I/flutter ( 3549): RangeError (index): Invalid value: Valid value range is empty: 0
I/flutter ( 3549): onCameraIdle#_lastMapPosition = LatLng(***, ***)
I/flutter ( 3549): package:google_map_location_picker/src/map.dart 284:7 | GeolocationStatus.granted
I/flutter ( 3549): RangeError (index): Invalid value: Valid value range is empty: 0

```
![Screenshot_20191021-221135](https://user-images.githubusercontent.com/53362329/67225304-a0921500-f450-11e9-9c7d-9f9dde5b94c7.png)

**Comment by humazed** [comment]
possible duplicate of #14

**Comment by humazed** [comment]
I'm closing the current one as a duplicate.
If you disagree please write in the comments
and I will reopen it.

**Comment by badaween-ahmed** [comment]
Sorry it was my fault. I read that but didn't give  Places, Geolocation and Geocoding API. But is this okay if I am still getting some exception

```
I/flutter (10962): onCameraMoveStarted#_lastMapPosition = LatLng(18.570053727657285, 42.768663950264454)
I/flutter (10962): onCameraIdle#_lastMapPosition = LatLng(18.57033849411387, 42.76811745017767)
I/flutter (10962): package:google_map_location_picker/src/map.dart 284:7 | GeolocationStatus.granted
I/flutter (10962): RangeError (index): Invalid value: Valid value range is empty: 0

```

**Comment by humazed** [comment]
The only error I see is 
` RangeError (index): Invalid value: Valid value range is empty: 0`

and I'm not sure it's from the library

are you get it using the sample app only?

**Comment by badaween-ahmed** [comment]
I am running the sample app. And I just notice I can search but when I tap nothing is happening also not getting the live update like shown in the Gif.

![Screenshot from 2019-10-21 23-19-14](https://user-images.githubusercontent.com/53362329/67229676-6d07b880-f459-11e9-8c65-b9fd522e410b.png)

**Comment by humazed** [comment]
there are 3 places to but the api key in,
in dart code, in android, and in ios native.

**Comment by badaween-ahmed** [comment]
I am not using iPhone so I didn't add api key there. I added inside AndroidManifest.xml and in main.dart I sent these screenshots if I am doing anything wrong. I was trying to debug and when I run on the emulator I succeded to search and navigate to that location but still no update 'Unnamed place' all the time and so many exceptions. I will make a dummy project tomorrow and let you know.

![sdasd](https://user-images.githubusercontent.com/53362329/67237211-74828e00-f468-11e9-8a8f-484b9f8d3f59.png)
![sdfafsdf](https://user-images.githubusercontent.com/53362329/67237227-7e0bf600-f468-11e9-84f2-f9ddc33e73e9.png)
